CONTRIBUTING.md

```diff
@@ -20,9 +20,9 @@ Please update the [docs](README.md) accordingly so that there are no discrepanci
 
 ## Developing
 
-- `grunt test` run the jasmine and mocha tests
-- `grunt build` run webpack and bundle the source
-- `grunt version` prepare the code for release
+- `npm run test` or `grunt test` run the jasmine and mocha tests
+- `npm run build` or `grunt build` run webpack and bundle the source
+- `npm run version` or `grunt version` prepare the code for release
```

Comment:
Can we follow the `grunt test` and confirm the results?

Reply:
If you want to use grunt, you need some extra configuration to the codebase, checkout [Grunt Docs](https://gruntjs.com/getting-started). I can do that, but it should work absolutely fine with npm itself, which wasn't mentioned in the CONTRIBUTING.md file. Do I need to add grunt too? Or have I understood your issue wrong, please correct me if so.